### ISSUE TITLE
Дата расторжения в договарах и актах

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2724]
Originally reported on Google Code with ID 2724

```
В документе Договора и акты (Отдел кадров)при заполнении даты расторжения договора появилась
ошибка: Введена неправильная дата.
По хронологии все выдержано правильно: Дата расторжения позже даты начала. 
Лунинецлес.

Выполнено Н.Садовской. 
 if DateBegin <= Self.FieldByName("usr$datebegin").AsDateTime then
        call Exception.Raise("exception", "Введена неправильная дата!")
      end if
Знак "<" изменили на ">".
 Настройка приложена.

```

Reported by `IrinaSpakoV` on 2012-03-12 07:05:05

<hr>
- _Attachment: [lnc_2012_03_12 Дата расторжения в договорах и актах.gsf](https://storage.googleapis.com/google-code-attachments/gedemin/issue-2724/comment-0/lnc_2012_03_12 Дата расторжения в договорах и актах.gsf)_

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2012-03-12 16:05:38
- **Labels added**: Type-Defect, Priority-High, Wage

**Comment by gsbelarus** [comment]
Reported by `roberto1813` on 2013-05-13 14:13:45
- **Status changed**: `Fixed`

**Comment by gsbelarus** [comment]
Reported by `roberto1813` on 2013-08-23 12:13:11
- **Status changed**: `Duplicate`
- **Merged into**: #3223
